<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewShapeFileDialogForm</class>
 <widget class="QDialog" name="NewShapeFileDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>351</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New ShapeFile Layer...</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>329</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Type</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="m_radioBtnPoint">
       <property name="text">
        <string>Point</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_radioBtnLine">
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_radioBtnPolygon">
       <property name="text">
        <string>Polygon</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>72</y>
     <width>329</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="m_srsEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>WGS 84, SRID: 4326</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_specifySRSButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Specify SRS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>134</y>
     <width>329</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Attributes List</string>
   </property>
   <widget class="QTableWidget" name="m_tableAttributesWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Width</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>FID</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Integer</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>name</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>String</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>descr</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>String</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>80</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="m_buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>329</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewShapeFileDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewShapeFileDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
